cmd/apiserver/app/server/options.go

```diff
@@ -31,16 +31,16 @@ type ServiceCatalogServerOptions struct {
 	StorageTypeString string
 	// the runtime configuration of our server
 	GenericServerRunOptions *genericserveroptions.ServerRunOptions
+	// the admission options
+	AdmissionOptions *genericserveroptions.AdmissionOptions
```

Comment:
@pmorie Yes, now they need to be explicitly declared separately from ServerRunOptions